client/landing/jetpack-cloud/components/backup-activity-log/backup-activity-type-selector/placeholder.tsx

```diff
@@ -0,0 +1,23 @@
+/**
+ * External dependencies
+ */
+import React, { FunctionComponent } from 'react';
+import { useTranslate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+
+import { Button } from '@automattic/components';
+
+const BackupsActivityTypeSelectorPlaceholder: FunctionComponent = () => {
+	const translate = useTranslate();
+
+	return (
+		<Button className="backup-activity-type-selector__placeholder" disabled>
+			{ translate( 'Activity type' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **24** times:
`translate( 'Activity Type' )` *ES Score: 10*